src/main/resources/application.properties

```diff
@@ -1 +1,21 @@
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://localhost:3306/pupdb
+spring.datasource.username=
+spring.datasource.password=
 
+spring.profiles.active=dev
+
+spring.servlet.multipart.max-file-size=1024KB
+spring.servlet.multipart.max-request-size=1024KB
+
+spring.kafka.consumer.bootstrap-servers=localhost:9092
+spring.kafka.consumer.group-id=myplantdiary
+spring.kafka.consumer.auto-offset-reset=earliest
+spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+spring.kafka.consumer.value-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+
+spring.kafka.producer.bootstrap-servers=localhost:9092
+spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
+spring.kafka.producer.value-serializer=org.apache.kafka.common.serialization.StringSerializer
+
+// this code will allow you to connect to the database, this is just a draft code you will need connection attributes for it to connect. 
```

Comment:
Do not put coversation in source code - that's a no-no!  Conversation belongs only in the pull reqeuest.